framework/lib/components/tags-input/tags-input.tsx

```diff
@@ -0,0 +1,170 @@
+import React, {
```

Comment:
The component needs unit tests.

Reply:
Lmao, it is a complement when Oles reviews your PR and the almost  only thing he comments on is to add unit tests.